src/decoder_wildmidi.cpp

```diff
@@ -272,7 +297,12 @@ WildMidiDecoder::WildMidiDecoder(const std::string file_name) {
 	}
 	Output::Debug("WildMidi: Using %s as configuration file...", config_file.c_str());
 
+#if LIBWILDMIDI_VERSION >= 1027 // at least 0.4.3
```

Comment:
Do we need to support old wildmidi still?

Reply:
This feature will be only useful when you want to load instrument files from a non-filesystem path (e.g. zip or APK) and it was actually submitted by CBreiti for us.
So there is no reason to reject older wildmidi Versions here as they will still work perfectly fine.